{
  "uid" : "ac3dba2360185aee",
  "name" : "doesn't revert on transfer",
  "fullName" : "Contract: SafeERC20 with token that returns no boolean values.doesn't revert on transfer",
  "historyId" : "e3f1449356773a7b9dc3ca89b735d94",
  "time" : {
    "start" : 1635555008425,
    "stop" : 1635555009431,
    "duration" : 1006
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067e820>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067e820>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/utils/SafeERC20.test.js:37:77)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: SafeERC20 with token that returns no boolean values"
  }, {
    "name" : "testClass",
    "value" : "Contract: SafeERC20 with token that returns no boolean values"
  }, {
    "name" : "testMethod",
    "value" : "doesn't revert on transfer"
  }, {
    "name" : "package",
    "value" : "Contract: SafeERC20 with token that returns no boolean values"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "627c92e8dc252d42",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0681d90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554943385,
        "stop" : 1635554944380,
        "duration" : 995
      }
    }, {
      "uid" : "1f5480816d17a524",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0506790>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554922250,
        "stop" : 1635554922837,
        "duration" : 587
      }
    }, {
      "uid" : "d09cfbf266d32b63",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067cdf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554911135,
        "stop" : 1635554911723,
        "duration" : 588
      }
    }, {
      "uid" : "4ab1cd3569e9a7bc",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17deaf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554906460,
        "stop" : 1635554907047,
        "duration" : 587
      }
    }, {
      "uid" : "f0cd1b9e307fba38",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a07caac0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554872097,
        "stop" : 1635554873084,
        "duration" : 987
      }
    }, {
      "uid" : "8557adb25677f68a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a06814f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554842378,
        "stop" : 1635554842966,
        "duration" : 588
      }
    }, {
      "uid" : "6b6e8451f52c46da",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067c700>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554829639,
        "stop" : 1635554830290,
        "duration" : 651
      }
    }, {
      "uid" : "3f59a13aed7033f7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be849d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554826444,
        "stop" : 1635554827036,
        "duration" : 592
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "6a4f98b8778714e4",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/6a4f98b8778714e4",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551255823,
          "stop" : 1635551255887,
          "duration" : 64
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ac3dba2360185aee.json",
  "parameterValues" : [ ]
}